### PR TITLE
apps: Fix some typos found by codespell

### PR DESCRIPTION
**Pull request by stweil**

### CODE REVIEW

**Comment by stweil** [comment]
@owncloud-bot: My contribution is MIT licensed.

File: apps/files_external/3rdparty/Dropbox/Exception/RequestToken.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
  */
 
 /**
- * This exception is thrown when an error occured during the request_token process.
```
Comment (DeepDiver1975) [inline apps/files_external/3rdparty/Dropbox/Exception/RequestToken.php]: please dont change 3rdparty code

File: apps/files_external/3rdparty/Dropbox/OAuth.php [hunk 2]
```diff
@@ -62,7 +62,7 @@
     protected $oauth_token_secret = null;
 
     /**
-     * Get OAuth request last responce
+     * Get OAuth request last response
```
Comment (DeepDiver1975) [inline apps/files_external/3rdparty/Dropbox/OAuth.php]: please dont change 3rdparty code

File: apps/files_external/3rdparty/aws-sdk-php/Aws/S3/ResumableDownload.php [hunk 3]
```diff
@@ -150,7 +150,7 @@ protected function getRemaining()
         $targetByte = $this->meta['ContentLength'] - 1;
         $this->params['Range'] = "bytes={$current}-{$targetByte}";
 
-        // Set the starting offset so that the body is never seeked to before this point in the event of a retry
+        // Set the starting offset so that the body is never sought to before this point in the event of a retry
```
Comment (DeepDiver1975) [inline apps/files_external/3rdparty/aws-sdk-php/Aws/S3/ResumableDownload.php]: please dont change 3rdparty code

**Comment by DeepDiver1975** [comment]
besides the commants :+1:

**Comment by stweil** [comment]
@DeepDiver1975, fixes for directory 3rdparty are now removed from the pull request.

**Comment by icewind1991** [comment]
:-1:

**Comment by Xenopathic** [comment]
:+1:

**Pull request merged**